migrations/sql/V2019.05.02.06.01__add_dangerous_occurrence_code_subsections.sql

```diff
@@ -0,0 +1,59 @@
+    ('HSRCM', '1', '7', '3', '(3)', 'Mine hoisting plant accident', 'Any accident involving a mine hoisting plant and including sheaves, hoisting rope, shaft conveyance, shaft, shaft timber, or headframe structure', '1970-01-01', '9999-12-31', 'system-mds', 'system-mds'),
+    ('HSRCM', '1', '7', '3', '(4)', 'Inrush of water or debris', 'Unexpected inrush of water, mud, slurry, or debris', '1970-01-01', '9999-12-31', 'system-mds', 'system-mds'),
+    ('HSRCM', '1', '7', '3', '(5)', 'Explosion', 'Premature or unexpected explosion of explosives, gas or any dust', '1970-01-01', '9999-12-31', 'system-mds', 'system-mds'),
+    ('HSRCM', '1', '7', '3', '(6)', 'Release of dangerous gas', 'Significant inflow or release of explosive or other dangerous gas', '1970-01-01', '9999-12-31', 'system-mds', 'system-mds'),
+    ('HSRCM', '1', '7', '3', '(7)', 'Stoppage of ventilation', 'Unplanned stoppage of the main underground ventilation system', '1970-01-01', '9999-12-31', 'system-mds', 'system-mds'),
+    ('HSRCM', '1', '7', '3', '(8)', 'Out of control vehicle', 'A mine vehicle going out of control', '1970-01-01', '9999-12-31', 'system-mds', 'system-mds'),
+    ('HSRCM', '1', '7', '3', '(9)', 'Fire', 'Outbreak of fire if it endangers persons or threatens or damages equipment and all underground fires', '1970-01-01', '9999-12-31', 'system-mds', 'system-mds'),
+    ('HSRCM', '1', '7', '3', '(10)', 'Electrical equipment failure', 'Electrical equipment failure or incident that causes or threatens to cause injury to persons or damage to equipment or property', '1970-01-01', '9999-12-31', 'system-mds', 'system-mds'),
+    ('HSRCM', '1', '7', '3', '(11)', 'Any other unusual accident', 'Any other unusual accident or unexpected event which had the potential to result in serious injury', '1970-01-01', '9999-12-31', 'system-mds', 'system-mds')
+ON CONFLICT DO NOTHING;
+ 
+CREATE TABLE IF NOT EXISTS mine_incident_determination_type
+(
+    mine_incident_determination_type_code   character varying(3) PRIMARY KEY,
+    description                             character varying(100) NOT NULL,
+    display_order                           integer NOT NULL,
+    active_ind                              boolean NOT NULL DEFAULT 'true',
+
+    create_user                             character varying(60)                    NOT NULL,
+    create_timestamp                        timestamp with time zone DEFAULT now()   NOT NULL,
+    update_user                             character varying(60)                    NOT NULL,
+    update_timestamp                        timestamp with time zone DEFAULT now()   NOT NULL
+);
```

Comment:
Out of curiosity, was this discussed as being implemented as a nullable boolean?